src/ui/controls/shortcut/index.tsx

```diff
@@ -18,6 +18,18 @@ interface ShortcutLinkProps {
 export default function ShortcutLink(props: ShortcutLinkProps) {
     const cls = mergeClass('shortcut', props.class);
     const shortcut = props.shortcuts[props.commandName];
+    const shortcutClass = document.getElementsByClassName("shortcut");
```

Comment:
Hi, thanks for the provided solution. However it looks like a hack. The `ShortcutLink` class should know nothing about its implementations.

Reply:
From my knowledge props.shortcuts just stored the shortcuts to toggle the site, not the node.content.
Notice this behaviour 
![image](https://user-images.githubusercontent.com/49354833/69454626-9a83a280-0d34-11ea-8140-20b488d46dc7.png)
I added a test message to the end of the line
![image](https://user-images.githubusercontent.com/49354833/69454681-b5eead80-0d34-11ea-8ac1-42fe8c008737.png)

What I assume it did 
1.set the text of the node  "Toggle Extension\nCtrl+D"
2.append the initial "Toggle Extension\nCtrl+Dtest"

String is now - "Toggle Extension\nCtrl+DToggle Extension\nCtrl+Dtest"

1. inital = node.content 
1.set the text of the code "Toggle Extension\nCtrl+D"
2. append the initial "Toggle Extension\nCtrl+DToggle Extension\nCtrl+Dtest" + "test" 

String is now "Toggle Extension\nCtrl+DToggle Extension\nCtrl+DToggle Extension\nCtrl+Dtesttest"

Refer to my comment below